index.html

```diff
@@ -6281,7 +6336,43 @@ <h3><dfn>Execute Async Script</dfn></h3>
       and data <var>result</var>.
 </ol>
 </section> <!-- /Execute Async Script -->
-</section> <!-- /Executing Script -->
+<section>
+ <h3><dfn>Pin Script</dfn></h3>
+
+ <table class="simple jsoncommand">
+  <tr>
+   <th>HTTP Method</th>
+   <th>URI Template</th>
+  </tr>
+  <tr>
+   <td>POST</td>
+   <td>/session/{<var>session id</var>}/execute/pin/{<var>name</var>}</td>
+  </tr>
+ </table>
+
+ <p>The <a>remote end steps</a> are:
+
+ <ol>
+  <li><p>Let <var>script</var> be the result of
+ <a>getting a property</a> named <var>script</var>
+ from the <var>parameters</var>.
+
+   <li><p>If <var>script</var> is <a>undefined</a> or is not a <a>String</a>,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+   <li><p>If the <a>current browsing context</a> is <a>no longer
+ open</a>, return <a>error</a> with <a>error code</a> <a>no such
+ window</a>.
```

Comment:
I think we should drop this requirement for pinning scripts as it can be done globally.

Reply:
> I do see value in a driver refusing to pin a script for whatever reason deemed necessary

Is this the same as the overwrite/error discussion, or is this just a failsafe for a driver to punt on things if it wants?

Reply:
> > I do see value in a driver refusing to pin a script for whatever reason deemed necessary
> 
> Is this the same as the overwrite/error discussion, or is this just a failsafe for a driver to punt on things if it wants?

Yes, this is specifically about the fact that something might happen on the driver side that might prevent it from storing the script. An example would be the driver running out of memory.

I thought we were all in agreement that another script pinned under the same name should overwrite the last one. At least I don’t think _I_ suggested that.